scripts/cluster_pdb_val_mmcifs.py

```diff
@@ -1160,7 +1160,7 @@ def filter_to_low_homology_sequences(
         filtered_interface_df = interface_clusters.filter(
             pl.col("interface_type") == interface_type
         )
-        if sample_size is not None:
+        if exists(sample_size):
```

Comment:
🤣

Reply:
Your coding style has been a bad influence on me :upside_down_face:

Reply:
indeed, it isn't traditional